README.md

```diff
@@ -51,7 +51,7 @@ The following are the default values, only specify values if they differ from th
 webSecurity:
   cors:
     allowCredentials: true
-    allowedHeaders: "X-Requested-With,Content-Type,Accept,Origin"
+    allowedHeaders: "X-Requested-With,Content-Type,Accept,Origin,Authorization"
```

Comment:
sure! :) it came with that funky ordering from the `CrossOriginFilter` values. sorting it seems sane though - easier to spot a specific header.